### PR TITLE
KHP3-6250: Patient Selection Should Redirect to Case Management on the Patient Chart Instead of Home

### PR DESCRIPTION
**Pull request by its-kios09**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
see: https://thepalladiumgroup.atlassian.net/browse/KHP3-6250

## Screenshots
case-update

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-patient-clinical-view-app/src/case-management/table/case-management-list-active-component.tsx [hunk 1]
```diff
@@ -37,14 +37,13 @@ const CaseManagementListActive: React.FC<CaseManagementListActiveProps> = ({ set
 
   const { data: activeCasesData, error: activeCasesError } = useActivecases(caseManagerPersonUuid);
 
-  const patientChartUrl = '${openmrsSpaBase}/patient/${patientUuid}/chart/Patient%20Summary';
+  const patientChartUrl = '${openmrsSpaBase}/patient/${patientUuid}/chart/case-encounter';
```
Comment (ojwanganto) [inline packages/esm-patient-clinical-view-app/src/case-management/table/case-management-list-active-component.tsx:40]: we should find a suitable name for case-encounter. May be case-management-history but i know that's long. case-managment-
encounters also sounds great

Comment (its-kios09) [reply packages/esm-patient-clinical-view-app/src/case-management/table/case-management-list-active-component.tsx:40]: Okay sure lemme update it same to the serial number column removal

**Comment by ojwanganto** [comment]
Thanks, @its-kios09. I have observed from the attached video that one has to keep tying the form's selection to eventually launch forms in the workspace. Kindly investigate and address the issue

**Comment by its-kios09** [comment]
> Thanks, @its-kios09. I have observed from the attached video that one has to keep tying the form's selection to eventually launch forms in the workspace. Kindly investigate and address the issue

That's the peek application behaves that way when you select on a combox so which makes me use the arrow buttons for selection

**Comment by ojwanganto** [comment]
cc @donaldkibet @makombe. Let's work to have this PR in

**Review by donaldkibet — APPROVED** [review]
Thanks @its-kios09

**Pull request merged**